builds/jenkins-delete-resources/Jenkinsfile

```diff
@@ -0,0 +1,554 @@
+import groovy.json.JsonSlurper
```

Comment:
Does this a different formatter than the other JenkinsFiles? Please use tabs/spaces consistently.

Reply:
done